src/View/AppView.php

```diff
@@ -51,13 +52,18 @@ public function initialize(): void
         $this->loadHelper('Time');
         $this->loadHelper('Url');
         $this->loadHelper('BEdita/I18n.I18n');
-        $this->loadHelper('Chialab/Rna.Rna');
         $this->loadHelper('Chialab/FrontendKit.Metadata');
         $this->loadHelper('Chialab/FrontendKit.DateRanges');
         $this->loadHelper('Chialab/FrontendKit.Download');
         $this->loadHelper('Chialab/FrontendKit.Placeholders');
         $this->loadHelper('Chialab/FrontendKit.Placeholders');
 
+        try {
+            $this->loadHelper('Chialab/Rna.Rna');
+        } catch (Exception) {
```

Comment:
We should be more strict about the exception being thrown:

```suggestion
        } catch (MissingHelperException) {
```

Its FQCN is: `\Cake\View\Exception\MissingHelperException`.